maven-core/src/main/java/org/apache/maven/exception/DefaultExceptionHandler.java

```diff
@@ -149,7 +149,8 @@ private ExceptionSummary handle( ProjectBuildingResult result )
         }
 
         String message = System.lineSeparator()
-            + "The project " + result.getProjectId() + " (" + result.getPomFile() + ") has "
+            + "The project " + ( result.getProjectId().isEmpty() ? "" : result.getProjectId() + " ")
```

Comment:
Checkstyle fails on this line. The last ")" has to be prefixed with a space.

Reply:
Already fixed. Thanks!